### ISSUE TITLE
Mixed transport

### ISSUE THREAD

**Issue by khenderick** [openvstorage/framework#875]
Mixed transport

**Comment by khenderick** [comment]
Fixed by #874, packaged in openvstorage-2.7.3-rev.3933.d183251

**Comment by kinvaris** [comment]
PASSED: 

Enabled RDMA

volumedriver:

```
In [4]: Configuration.get('/ovs/vpools/be4b7f66-b4a5-4e46-9634-d850b5477547/hosts/myvpoolwfgmpzrl0slo89w8/config')['backend_connection_manager']
Out[4]: 
{u'alba_connection_host': u'172.22.1.30',
 u'alba_connection_port': 26204,
 u'alba_connection_preset': u'rorapreset',
 u'alba_connection_rora_manifest_cache_capacity': 17179869184,
 u'alba_connection_timeout': 15,
 u'alba_connection_transport': u'TCP',
 u'alba_connection_use_rora': True,
 u'backend_interface_retries_on_error': 5,
 u'backend_interface_retry_backoff_multiplier': 2.0,
 u'backend_interface_retry_interval_secs': 1,
 u'backend_type': u'ALBA'}
```

proxy:

```
In [3]: Configuration.get('/ovs/vpools/be4b7f66-b4a5-4e46-9634-d850b5477547/proxies/d698b9d8-f226-4eec-856f-bbfc1b06e922/config/main')
Out[3]: 
{u'albamgr_cfg_url': u'arakoon://config/ovs/vpools/be4b7f66-b4a5-4e46-9634-d850b5477547/proxies/d698b9d8-f226-4eec-856f-bbfc1b06e922/config/abm?ini=%2Fopt%2FOpenvStorage%2Fconfig%2Farakoon_cacc.ini',
 u'fragment_cache': [u'none'],
 u'ips': [u'172.22.1.30'],
 u'log_level': u'info',
 u'manifest_cache_size': 17179869184,
 u'port': 26204,
 u'transport': u'tcp'}
```
